src/participant/prompts/exportToLanguage.ts

```diff
@@ -0,0 +1,36 @@
+import { PromptBase, type PromptArgsBase } from './promptBase';
+import type { UserPromptResponse } from './promptBase';
+
+export interface ExportToLanguagePromptArgs extends PromptArgsBase {
+  language: string;
+  includeDriverSyntax: boolean;
+}
+
+export class ExportToLanguagePrompt extends PromptBase<ExportToLanguagePromptArgs> {
+  protected getAssistantPrompt({
+    language,
+  }: ExportToLanguagePromptArgs): string {
+    return `You are a MongoDB expert.
+Your task is to convert a MongoDB playground script to the ${language} language.
+Take a user prompt as an input string and translate it to the target language.
+If the user specified to include import statements, add required imports at the beginning of the transpiled code.
+If the user specified to not include import statements, do not add any imports at the beginning of the transpiled code.
```

Comment:
How do you feel about conditionally showing only one of these to keep the prompt shorter with fewer instructions? 

If the user specified to include import statements then we have a line saying include import statements, and if they set it not to then we say don't include import statements.

Same with the driver syntax.

Reply:
I actually removed the import statements switch and forgot to update the prompt. Great that you pointed to this, thx! The model includes import statements by default when you include the driver syntax. I will update the text!

Reply:
Actually, it doesn't perform well without specifying booth options in the assistant message. The model then adds all helpers anyway, even if you say not to do it. It looks like when you explicitly list both available options and describe the expected behavior, only then the model properly gets the instruction. I refactored it a bit to combine the driver syntax and import statements cases.